sigstore-java/src/main/java/dev/sigstore/fulcio/client/FulcioVerifier.java

```diff
@@ -59,15 +60,26 @@ public static FulcioVerifier newFulcioVerifier(byte[] fulcioRoot, byte @Nullable
     // encountered in verifyCertPath
     new PKIXParameters(Collections.singleton(fulcioRootTrustAnchor));
 
-    return new FulcioVerifier(fulcioRootTrustAnchor, ctfePublicKeyObj);
+    return new FulcioVerifier(fulcioRootTrustAnchor, ctfePublicKeyObjs);
   }
 
-  private FulcioVerifier(TrustAnchor fulcioRoot, @Nullable PublicKey ctfePublicKey) {
+  private FulcioVerifier(TrustAnchor fulcioRoot, @Nullable List<PublicKey> ctfePublicKeys) {
     this.fulcioRoot = fulcioRoot;
-    if (ctfePublicKey != null) {
-      var ctLogInfo = new CTLogInfo(ctfePublicKey, "fulcio ct log", "unused-url");
+    if (ctfePublicKeys != null) {
+      var logInfos = new ArrayList<CTLogInfo>();
+      for (var pk : ctfePublicKeys) {
+        var ctLogInfo = new CTLogInfo(pk, "fulcio ct log", "unused-url");
+        logInfos.add(ctLogInfo);
+        System.out.println(Base64.getEncoder().encodeToString(ctLogInfo.getID()));
```

Comment:
do you mean to include this system out call?

Reply:
yup... nope